main.py

```diff
@@ -0,0 +1,19 @@
+# Author: Yehuda Shani
+# Date: 20/5/2021
+# Program description: This program is the main program that runs the GPT-3 model on powerpoint presentations.
+# and writes the summary to a JSON file.
```

Comment:
These lines of code are not neede when you have version control, they are done automatically